### DISCUSSION TITLE
metadata支持的内容

### DISCUSSION THREAD

**Discussion by arnodjiang** [obsidianzh/forum#77] [General综合]
在实践中我使用了以下格式：

```md
---
aliases: [AI, dl]
tags: ["#dl", "#ml"]
---
```

![image](https://user-images.githubusercontent.com/59393870/113496328-2ca3d980-952b-11eb-9536-6e0fb5c9b93c.png)

在预览中是支持的，但请问还支持其他 YAML 的 metadata 吗？比如添加 title，date等字段

我在添加日期字段后，预览无法显示，仍然只有以上两个字段的内容，obsidian是否还支持其他的 metadata?

```md
date: [2021-02-18]
```

**Comment by dreamkidd** [comment]
现在官方只支持一下三个标签，其他的都属于自定义标签

https://help.obsidian.md/Advanced+topics/YAML+front+matter

Currently three keys are natively supported: `tags`, `aliases` and `cssclass`
